Artifacts/Get-BCArtifactUrl.ps1

```diff
@@ -188,99 +192,124 @@ try {
                 $version = "$($version.TrimEnd('.'))."
             }
             $GetListUrl += "&prefix=$($Version)"
+            $upMajorFilter = $version.Split('.')[0]
+            $upVersionFilter = $version.Substring($version.Length).TrimStart('.')
         }
-        
-        $Artifacts = @()
-        $nextMarker = ''
-        $currentMarker = ''
-        $downloadAttempt = 1
-        $downloadRetryAttempts = 10
-        do {
-            if ($currentMarker -ne $nextMarker)
-            {
-                $currentMarker = $nextMarker
-                $downloadAttempt = 1
+
+        if ($bcContainerHelperConfig.useUniversalPackagesForArtifacts) {
+            $upArr = $bcContainerHelperConfig.useUniversalPackagesForArtifacts.Split('/')
+            $feedApiUrl = "https://feeds.dev.azure.com/$($upArr[0])/$($upArr[1])/_apis/packaging/feeds/$($storageAccount.Split('.')[0])"
```

Comment:
Split by `/` only to join by `/`? Some code comments around here would be useful.

Reply:
:-)